doc/reference/storage_btrfs.md

```diff
@@ -58,14 +58,14 @@ btrfs.mount\_options            | string    | user\_subvol\_rm\_allowed  | Mount
 source                          | string    | -                          | Path to block device or loop file or file system entry
 
 ### Storage volume configuration
-Key                     | Type      | Condition                 | Default                               | Description
-:--                     | :---      | :--------                 | :------                               | :----------
-security.shifted        | bool      | custom volume             | false                                 | {{enable_ID_shifting}}
-security.unmapped       | bool      | custom volume             | false                                 | Disable ID mapping for the volume
-size                    | string    | appropriate driver        | same as volume.size                   | Size/quota of the storage volume
-snapshots.expiry        | string    | custom volume             | -                                     | {{snapshot_expiry_format}}
-snapshots.pattern       | string    | custom volume             | snap%d                                | {{snapshot_pattern_format}}
-snapshots.schedule      | string    | custom volume             | -                                     | {{snapshot_schedule_format}}
+Key                     | Type      | Condition                 | Default                                     | Description
+:--                     | :---      | :--------                 | :------                                     | :----------
+security.shifted        | bool      | custom volume             | false                                       | {{enable_ID_shifting}}
+security.unmapped       | bool      | custom volume             | false                                       | Disable ID mapping for the volume
```

Comment:
Should these be "same as volume.security.* or false"?

Reply:
I don't think those are supported as volume defaults in the code, so the docs are correct technically.
However these options do stick out a bit, and I'm wondering if there is any reason why we shouldn't expand the code to also allow these options to have their defaults specified at the pool level too. Any objections @stgraber

Reply:
I'd be happy to merge this as-is (once the tests are fixed), and we can always expand the supported keys later. They are not supported today as pool level defaults, so it wouldn't be a regression.

Reply:
@presztak are you OK to make this fields like the other `volume.*` fields?

Reply:
Done.